frontend/awx/main/routes/useAwxManagementJobsRoutes.tsx

```diff
@@ -5,10 +5,8 @@ import { PageNavigationItem } from '../../../../framework';
 import { PageNotImplemented } from '../../../../framework/PageEmptyStates/PageNotImplemented';
 import { ManagementJobPage } from '../../administration/management-jobs/ManagementJobPage/ManagementJobPage';
 import { ManagementJobs } from '../../administration/management-jobs/ManagementJobs';
-import { awxAPI } from '../../common/api/awx-utils';
-import { SchedulePage } from '../../views/schedules/SchedulePage/SchedulePage';
-import { Schedules } from '../../views/schedules/Schedules';
 import { AwxRoute } from '../AwxRoutes';
+// import { ManagementJobSchedules } from '../../administration/management-jobs/ManagementJobPage/ManagementJobSchedules';
```

Comment:
Just one last thing can we remove this comment. I've been told we shouldn't push any commented code to main.

Reply:
done, thanks for pointing that out